src/calculation/Calculator.ts

```diff
@@ -518,44 +519,47 @@ export async function calculateDataRequirements(
     }
   });
 
-  // Only use "unique" retrieves
```

Comment:
I was about to throw a huge red flag on this code. But I noticed it was the removed code. 😆

We do want to look at ALL retrieves and how they are filtered. The hard part of this is that we may want to effectively "or" the requirements of the retrieves with matching resourceType and valueSet or code (aka. data element) together. But I think for now its okay if multiple dataRequirements are generated for the same "data element". That does effectively communicate the "or" scenario just fine.